tutorials/date-picker-data.md

```diff
@@ -58,29 +58,23 @@ locale is Chakma:
 use icu::locale::locale;
 use icu_provider_blob::BlobDataProvider;
 
-// Just below the imports (fill in the path):
-const CCP_BLOB_PATH: &str = "<absolute path to ccp.blob>";
```

Comment:
Nit: I think I prefer keeping the string out here to make it easier to see/find

Reply:
I inlined it because it's annoying to copy three code fragments to three different locations when following the tutorial. This also _might_ confuse Rust newbies who could think the path needs to be const.